File: the_hashtag_generator.js
function generateHashtag(str) {
  let result = "#";

  console.log(str);

  const strs = str.split(" ");
  for (let i = 0; i < strs.length; i++) {
    if (strs[i].length != 0) {
      result +=
        strs[i].length < 1
          ? strs[i].toUpperCase()
          : strs[i].charAt(0).toUpperCase() + strs[i].slice(1).toLowerCase();
    }
  }

  if (result.length > 140 || result == "#") return false;

  return result;
}

File: the_hashtag_generator.py
def generate_hashtag(s):
    result = "#"

    for i in s.split(" "):
        if (i != "" and len(i) > 1):
            result += i[0].upper() + i[1:].lower()

        elif len(i) == 1:
            result += i[0].upper()

    return False if (s == "" or len(result) > 140) else result
